src/util/InstanceCount.cc

```diff
@@ -18,8 +21,17 @@ LOG_LOGGER _log = LOG_GET("lsst.qserv.util.InstanceCount");
 
 namespace lsst::qserv::util {
 
-std::map<std::string, int> InstanceCount::_instances;
-std::recursive_mutex InstanceCount::_mx;
+InstanceCountData InstanceCount::_icData;
+
+InstanceCountData::InstanceCountData() {
+    std::cout << "InstanceCountData " << " mx=" << (void*)(&_mx) << " _inst=" << (void*)(&_instances)
```

Comment:
Do you really want to post these messages to `std::cout` instead of the LSST Logger stream?

Reply:
They need to go to `cout` as the instances are built statically and there's no guarantee that they will be called after `log` has been defined. Their only purpose is to detect that the object has been defined more than once, which was actually pretty useful as there wound up being multiple copies be made due to linker problems.